lib/resources/example_json/soarian_examples_document_reference_with_addtionaldoc.rb

```diff
@@ -0,0 +1,85 @@
+
+module Cerner
+  module Resources
+
+    SOARIAN_DOCUMENT_REFERENCE_DOCREF_BUNDLE_ATTACHMENT ||= {
```

Comment:
Looks like this can probably go in the same file as the other document reference example?